testsuite/tests/properties/find_all_references_syntax_errors/test.out

```diff
@@ -10,5 +10,6 @@ Analyzing pkg.ads
 #################
 
 References to Foo (pkg.ads, 2:14-2:17)
+WARNING: File dummy.ads not found
```

Comment:
Could you rather add a dedicated test for this? That way in the future we won’t mistakenly “fix” this test?